### PR TITLE
All Doc Changes for v0.25.0 Release

### PR DESCRIPTION
**Pull request by thedavidprice**
- [x] cookbook/self-hosting-redwood PR#552
- [x] docs/deploy PR#553 (<-- I don’t think this one is correct. On my list.)
- [x] docs/local-postgres PR#555
- [x] docs/cli-commands PR#550 (‘db’ and ’prisma sections)
- [x] TO DO: CLI-commands deploy section
- [x] cookbook/file-upload and docs/quick-start PR#554
- [x] /tutorial and /tutorial2 PR#549
- [x] /docs/authentication PR#543

### CODE REVIEW

File: docs/deploy.md [hunk 1]
```diff
@@ -29,32 +29,44 @@ Each hosting provider has different requirements for how (and where) the deploym
...
+```shell
+yarn rw prisma deploy <target>
+```
+
+For example:
+
 ```shell
-yarn rw build && yarn rw db up --no-db-client --auto-approve && yarn rw dataMigrate up
+# Build command for Netlify deploy target
+yarn rw prisma deploy netlify
```
Comment (dac09) [inline docs/deploy.md:42]:
```suggestion
yarn rw deploy netlify
```

File: docs/deploy.md [hunk 2]
```diff
@@ -29,32 +29,44 @@ Each hosting provider has different requirements for how (and where) the deploym
...
+```
+
+```shell
+# Build command for Vercel deploy target
+yarn rw prisma deploy vercel
```
Comment (dac09) [inline docs/deploy.md:47]:
```suggestion
yarn rw deploy vercel
```

File: docs/deploy.md [hunk 3]
```diff
@@ -29,32 +29,44 @@ Each hosting provider has different requirements for how (and where) the deploym
...
+```
+
+
+```shell
+# Build command for Serverless deploy target
+yarn rw prisma deploy serverless
```
Comment (dac09) [inline docs/deploy.md:53]:
```suggestion
yarn rw deploy serverless
```

File: docs/deploy.md [hunk 4]
```diff
@@ -29,32 +29,44 @@ Each hosting provider has different requirements for how (and where) the deploym
...
 datasource DS {
-  provider = ["sqlite", "postgresql"]
+  provider = "postgresql"
   url      = env("DATABASE_URL")
 }
 ```
 
 The `url` setting above accesses the database connection string via an environment variable, `DATABASE_URL`. Using env vars is the recommended method for both ease of development process as well as security best practices.
 
-Whenever you make changes to your `schema.prisma`, you must run the following commands:
```
Comment (dac09) [inline docs/deploy.md:50]: Shouldn't we still have this and just say you need to run yarn rw prisma migrate dev instead?

File: docs/deploy.md (hunk 3 above)
Comment (thedavidprice) [inline docs/deploy.md:53]:
```suggestion
# Build command for AWS Lambdas using the https://serverless.com framework
rw deploy aws serverless --side api
```

File: docs/deploy.md [hunk 5]
```diff
@@ -29,32 +29,44 @@ Each hosting provider has different requirements for how (and where) the deploym
...
-It is possible to use a different database type for local development and production. To do this, you need to use the Prisma dynamic provider syntax. For example, to use SQLite locally and PostgreSQL in production, include this in your `schema.prisma`:
 ```javascript
 datasource DS {
-  provider = ["sqlite", "postgresql"]
+  provider = "postgresql"
   url      = env("DATABASE_URL")
 }
 ```
 
 The `url` setting above accesses the database connection string via an environment variable, `DATABASE_URL`. Using env vars is the recommended method for both ease of development process as well as security best practices.
```
Comment (thedavidprice) [inline docs/deploy.md:69]:
```suggestion

Whenever you make changes to your `schema.prisma`, you must run the following command:
```shell	
$ yarn rw prisma migrate dev # creates and applies a new Prisma DB migration	
```	

_Don't forget to save, commit, and push any changes prior to deploying!_
```

**Pull request merged**